Frontend/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
```

Comment:
Please add also file for `production` mode. Ideally you should have 3 files:
- webpack.common.js (here you will place common modules for `development` and `production` mode)
- webpack.dev.js (`development` features only)
- webpack.prod.js (`production` features only)
https://webpack.js.org/guides/production/